cpp/starkiller/main.cpp

```diff
@@ -0,0 +1,180 @@
+#include <torch/torch.h>
+#include <torch/script.h> // One-stop header.
+
+#include <starkiller.H>
+
+#include <AMReX_ParmParse.H>
+#include <AMReX_PlotFileUtil.H>
+#include <AMReX_VisMF.H>
+
+#include <iostream>
+#include <memory>
+
+using namespace amrex;
+
+
+int main (int argc, char* argv[])
+{
+    amrex::Initialize(argc, argv);
+
+    {
+        int n_cell = 128;
+        int max_grid_size = 32;
+
+	std::string model_filename = "my_model.pt";
```

Comment:
ts_model.pt or could just come from probin, it looks like you meant to do that.

Reply:
Oh never mind you are doing that, this just needs to be deleted

Reply:
I'm getting an error that it can't open `my_model.pt` so it looks like the parmparse isn't working? When I change it manually to `ts_model.pt` it gives me a different error. 

```MPI initialized with 1 MPI processes                                                                                                                                              
MPI initialized with thread support level 0                                                                                                                                       
AMReX (21.06-43-gc797f6b6979a) initialized                                                                                                                                        
terminate called after throwing an instance of 'torch::jit::ErrorReport'                                                                                                          
  what():                                                                                                                                                                         
Unknown type name 'NoneType':                                                                                                                                                     
Serialized   File "code/__torch__/torch/nn/modules/linear.py", line 7                                                                                                             
  bias : Tensor                                                                                                                                                                   
  training : bool                                                                                                                                                                 
  _is_full_backward_hook : NoneType                                                                                                                                               
                           ~~~~~~~~ <--- HERE                                                                                                                                     
  out_features : Final[int] = 64                                                                                                                                                  
  in_features : Final[int] = 16                                                                                                                                                   
                                                                                                                                                                                  
SIGABRT                                                                                                                                                                           
See Backtrace.0 file for details                                                                                                                                                  
Rank 0 [Thu Aug 19 10:28:03 2021] [c4-0c1s12n3] application called MPI_Abort(MPI_COMM_WORLD, 6) - process 0                                                                       
Reading ts_model.pt file ... srun: error: nid00883: task 0: Aborted                                                                                                               
srun: launch/slurm: _step_signal: Terminating StepId=45831548.0                                                                                                                   
                                                                                                                                                                                  ```
Which looks like lib torch isn't loading everything correctly... That might be my fault.

Reply:
Sorry for the confusion. So `my_model.pt` is the pytorch model file that you gave me; `ts_model.pt` is the torch script version of that model. I would try to run `convert.py` to get `ts_model.py` from `my_model.pt` on your computer. It's possible that the version of pytorch and libtorch are different on your computer than on mine and that is what's causing the error.

Reply:
It'd also be a good idea to check which version of libtorch you have. It should match the pytorch version that you're using.

Reply:
Maybe it'll be less confusing if I remove `ts_model.pt` and add instructions to convert the model file in README.